app/src/main/java/net/gsantner/opoc/ui/FilesystemViewerAdapter.java

```diff
@@ -142,13 +149,29 @@ public void onBindViewHolder(@NonNull ilesystemViewerViewHolder holder, int posi
         holder.description.setText(!_dopt.descModtimeInsteadOfParent || holder.title.getText().toString().equals("..") ? descriptionFile.getAbsolutePath() : DateUtils.formatDateTime(_context, descriptionFile.lastModified(), (DateUtils.FORMAT_SHOW_TIME | DateUtils.FORMAT_SHOW_DATE | DateUtils.FORMAT_SHOW_YEAR | DateUtils.FORMAT_NUMERIC_DATE)));
         holder.description.setTextColor(ContextCompat.getColor(_context, _dopt.secondaryTextColor));
 
-        holder.image.setImageResource(file.isDirectory() ? _dopt.folderImage : _dopt.fileImage);
```

Comment:
You can keep this for all cases.
When the thumbnail is loading for example it's still fine to show the "file" icon